sdk/python/tests/compiler/component_builder_test.py

```diff
@@ -135,9 +135,22 @@ def sample_component_func_four() -> NamedTuple(
   output = namedtuple('output', ['a', 'b'])
   return output(1.0, 'test')
 
+
+def _func_to_entrypoint(component_func):
+  from kfp.components._python_op import _func_to_component_spec
+  component_spec = _func_to_component_spec(component_func)
+  command_line_args = component_spec.implementation.container.command
+
+  dash_c_index = command_line_args.index('-c')
+  program_code_index = dash_c_index + 1
+  program_code = command_line_args[program_code_index]
+  return program_code
+
+
 class TestGenerator(unittest.TestCase):
```

Comment:
Maybe we can add a test for a component function with default values?

Reply:
There are pretty extensive tests for default value handlingin the Lightweight components test suite. Do you have some additional scenarios in mind?